Cards/Cardstyle.css

```diff
@@ -0,0 +1,187 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700;900&display=swap');
+
+:root {
+    --primary-color : #c2004b;
+    --secondary-color : #eb679a;
+    --light-color : #ffeff5;
+    --dark-color : #72022d;
+}
+* {
+    margin: 0;
+    padding: 0;
+    font-family: Roboto;
+}
+::-webkit-scrollbar {
+    width: 0px;
+}
+body {
+    background-color: #F7F7F7;
+    width: 100%;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+/* Common css for card */
+.card {
+    border-radius: 4px;
+    width: 350px;
+    margin : 35px 0px;
+    display: flex;
+    flex-direction: column;
+    background-color: white;
+    overflow: hidden;
+}
+.card .card-container .card-text {
+    padding: 1rem;
+}
+.card .card-container .card-text .card-heading {
+    font-size : 1.25rem;
+    line-height: 2rem;
+}
+.card .card-container .card-text .card-subheading {
+    font-size: 0.875rem;
+    color: #0000008A;
+    line-height: 1.375rem;
+}
+.card .card-container .card-information {
+    padding: 1rem 1rem 0.5rem;
+    color: rgb(117 117 130);
+    font-size: 14px;
+    line-height: 1.25rem;
+}
+/* Common css for all card-action panel */
+.card-actions {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    background-color: white;
+    padding: 0.5rem;
+    border-bottom-left-radius: 4px;
+    border-bottom-right-radius: 4px;
+}
+
+.card-actions button,
+.card-actions div>i {
+    color: var(--primary-color);
+    border: 0px;
+    background-color: transparent;
+    border-radius: 4px;
+    cursor: pointer;
+}
+
+.card-actions .card-buttons button {
+    height: 36px;
+    font-weight: 600;
+    letter-spacing: .0892857143em;
```

Comment:
refrain from using `rem` for small measurements, instead use `px` upto 12px
Why such a specific decimal value for letter-spacing is used here?

Reply:
i actually took this value from component itself, that's why i kept the same